drivers/net/mvpp2/mrvl_ethdev.c

```diff
@@ -2654,6 +2672,20 @@ mrvl_rx_pkt_burst(void *rxq, struct rte_mbuf **rx_pkts, uint16_t nb_pkts)
 				mrvl_desc_to_ol_flags(&descs[i],
 						      mbuf->packet_type);
 
+		if (mrvl_enable_ts[mbuf->port] && (q->priv->tai != CLOCK_INVALID)) {
```

Comment:
maybe unlikely would be good here.

Reply:
I've hidden `mrvl_enable_ts[]` into ptp build unit and exposed this check as function and wrapped in `unlikely`

Reply:
that does not help here. The compiler can't see that unlikely.